music21/romanText/translate.py

```diff
@@ -1661,10 +1682,10 @@ def _test_ending_contents(
 
         empty_bars_with_repeats = textwrap.dedent('''
             Time Signature: 2/4
-            m1: I
-            m2: ||:
-            m3: :||
-            m4: ||: :||
+            m1 I
+            m2 ||:
+            m3 :||
+            m4 ||: :||
```

Comment:
Add tests that the repeated measures do have an RN.figure == 'I' token in it after doing above.